Added TOTP module with a module README to OpenCyb3r/modules

omsej opened the pull request:
## Components:
1. TOTP generator.
2. Secret key generator for the TOTP.
3. Keys are stored in an encrypted file and generating secret keys or TOTP's are password protected.
4. Added a README.md for the module.

omsej committed:
Added TOTP module with a module README to OpenCyb3r/modules

evanlutz725 reviewed (commented):
This is incredible! Truly above and beyond! Last request is to put something into main.py so someone could run it from the main script - either instruct them to /modules/TOTP to set it up so they can retrieve their OTPs or to retrieve from an account of their choice. Other than that, it looks rock solid. Please use option 3 in main.py as 2 is already being worked on for a hashing function. Thanks!

omsej committed:
Update main.py

Added TOTP module

omsej committed:
Update main.py

Key Changes: 1. Added TOTP Generator as option #3 in the main menu 2. Created a dedicated totp_tool() function with a submenu for:    - First Time Setup    - Generate New TOTP Key    - Get TOTP Code    - Return to Main Menu  3. Added directory management for proper file access to allow TOTP module to be accessible from main.py.

omsej committed:
Merge pull request #1 from omsej/main

omsej commented:
> This is incredible! Truly above and beyond! Last request is to put something into main.py so someone could run it from the main script - either instruct them to /modules/TOTP to set it up so they can retrieve their OTPs or to retrieve from an account of their choice. Other than that, it looks rock solid. Please use option 3 in main.py as 2 is already being worked on for a hashing function. Thanks!

Main.py updated
TOTP runs from within main.py rather than directing the user to /modules/TOTP

evanlutz725 committed:
Merge branch 'main' into feature/totp-module
